waku/v2/waku_discv5.nim

```diff
@@ -89,6 +116,12 @@ proc new*(T: type WakuDiscoveryV5,
         )
         builder.withWakuCapabilities(flags)
         builder.withMultiaddrs(multiaddrs)
+
+        if relayShard.isSome():
+          let res = builder.withWakuRelaySharding(relayShard.get())
```

Comment:
Perhaps a good idea to add `debug` log here to indicate that we're building ENR with relay shard information?

Reply:
Agreed